bridge/modules/bridge/module.yaml

```diff
@@ -1,5 +1,19 @@
 schema_version: 1
 
 name: bridge
+version: 1.3.0
+
+envs:
+  - name: "STRIMZI_HOME"
+    value: "/opt/strimzi"
+  - name: "COM_REDHAT_COMPONENT"
+    value: "amqstreams12-bridge-openshift-container"
+
+artifacts:
+  - md5: f72d542fe7a79c8be710ee6190562702
+    name: kafka-bridge.tar.gz
+  - md5: 8384576ef02897c5e36174444d73caf0
+    name: kafka-bridge-licenses.tar.gz
+
```

Comment:
cekit3 processes the content in the "image.yaml" file last, so at build time the "kafka" module we use does not have access to these artifiacts and envs listed in the image.yaml. For the kafka module scripts to have access to these artifacts, we have to declare them in the module.yaml file of the module which uses them.

We can set it now or later, whatever you prefer. I guess it makes most sense to keep it at 1.2.0 since this is technically a refactor PR

Reply:
After reading cekit doc I know about that :-)